pkg/workload/kv/kv.go

```diff
@@ -121,6 +125,13 @@ func (w *kv) Hooks() workload.Hooks {
 			}
 			return nil
 		},
+		PostLoad: func(sqlDB *gosql.DB) error {
+			if w.secondaryIndex {
+				_, err := sqlDB.Exec(`CREATE INDEX IF NOT EXISTS v_idx ON kv (v)`)
```

Comment:
Is this going to be backfilled in the background? If so, the impact of that will be pretty nondeterministic. Maybe that doesn't matter now, but perhaps call it out on the flag.